src/components/places/places-list/places-list.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import PlaceRow from './places-row/places-row'
+
+const StyledPlacesList = styled.table`
+  background: orange;
+  width: 50%;
+  table-layout: fixed;
+`
+
+const tableHeaders = ['Item', 'Day', 'Category']
+
+const PlacesList = ({ places }) => {
+  return (
+    <StyledPlacesList>
+      <tr>
+        {tableHeaders.map(header => (
+          <th>{header}</th>
+        ))}
+      </tr>
+      {places.map(place => (
+        <PlaceRow place={place} />
```

Comment:
W sumie jak w wiekszości `map`: React się nie pluje o brak `key`? :P

Reply:
jak dodam eslinta to zacznie 🤣